helpers/dateTime.js

```diff
@@ -139,12 +139,6 @@ export function getDateNoConversion(value) {
 	return new Date(parsed.year, parsed.month - 1, parsed.date, parsed.hours, parsed.minutes, parsed.seconds);
 }
 
-let dateTimeDescriptor = null;
-export function getDateTimeDescriptorShared(refresh) {
```

Comment:
I just noticed that [Nova is using this](https://github.com/Brightspace/nova/blob/main/src/app/main/views/activities/view-activity.js#L25) -- I'll switch that before merging this.